packages/react-app/src/pages/api/savedQuery/index.ts

```diff
@@ -32,6 +32,16 @@ export const handler = async (
 		break;
 	}
 
+	case 'DELETE': {
+		try {
+			const data = await service.deleteQuery(req.query.id as string);
+			res.status(200).json({ success: true, data });
```

Comment:
Doesn't look like the delete data returns anything useful, might want to just 204

Reply:
yea, that would be better